var/spack/repos/builtin/packages/py-torch-geometric/package.py

```diff
@@ -19,40 +19,54 @@ class PyTorchGeometric(PythonPackage):
     graphs as well as on 3D meshes or point clouds."""
 
     homepage = "https://github.com/rusty1s/pytorch_geometric"
-    url = "https://github.com/rusty1s/pytorch_geometric/archive/1.6.0.tar.gz"
+    pypi = "torch-geometric/torch_geometric-2.1.0.post1.tar.gz"
 
-    version("1.6.3", sha256="1907c0f5eb7ab8f6f79a7e43703579b39b514501dd956ed0fc3d33210bcbb4c2")
-    version("1.6.0", sha256="7d5231cdcc2ebd4444f406cbf1537eb49bf90ab6f446eaf1b7af5cdbe105f3c9")
+    version(
+        "2.1.0.post1", sha256="32347402076ccf60fa50312825178f1e3e5ce5e7b3b3a8b2729ac699da24525d"
+    )
+    version("1.6.3", sha256="347f693bebcc8a621eda4867dafab91c04db5f596d7ed7ecb89b242f8ab5c6a1")
+    version("1.6.0", sha256="fbf43fe15421c9affc4fb361ba4db55cb9d3c64d0c29576bb58d332bf6d27fef")
 
     variant("cuda", default=False, description="Enable CUDA support")
 
+    # setup.py
+    depends_on("python@3.7:", when="@2:", type=("build", "run"))
     depends_on("python@3.6:", type=("build", "run"))
     depends_on("py-setuptools", type="build")
-    depends_on("py-pytest-runner", type="build")
-    depends_on("py-torch", type=("build", "run"))
-    depends_on("py-numpy", type=("build", "run"))
     depends_on("py-tqdm", type=("build", "run"))
+    depends_on("py-numpy", type=("build", "run"))
     depends_on("py-scipy", type=("build", "run"))
-    depends_on("py-networkx", type=("build", "run"))
-    depends_on("py-scikit-learn", type=("build", "run"))
```

Comment:
You seem to have removed py-scikit-learn and not added it anywhere else.  I assume this was intended?